docs/getting-started/ios/fastlane-swift.md

```diff
@@ -42,6 +42,84 @@ Run `fastlane <laneName>` in your terminal to execute _fastlane_.
 
 🎉
 
+## Get Started (SPM) (Beta)
+
+Swift Package Manager support is included as a preliminary feature that allows the usage of _fastlane_ with other SPM packages.
+
+### Step 1
+
+Create an executable Swift Package Manager project with `swift package init --type executable`.
+
+### Step 2
+
+Add the _fastlane_ dependency to your `Package.swift`.
+
+```swift
+.package(name: "Fastlane", url: "https://github.com/fastlane/fastlane", from: "2.179.0")
+```
+
+A full example of a working package description would be the following.
+
+```swift
+// swift-tools-version:5.2
+
+import PackageDescription
+
+let package = Package(
```

Comment:
@TimOliver I have updated the step 2 to give a full working example of a package description, see if this helps a bit ❤️